packages/core/admin/admin/src/components/GuidedTour/Tours.tsx

```diff
@@ -32,7 +32,7 @@ const tours = {
     {
       name: 'Introduction',
       content: (Step) => (
-        <Step.Root side="bottom" withArrow={false}>
+        <Step.Root side="bottom" sideOffset={33} withArrow={false}>
```

Comment:
how do you get to 33?

Reply:
the designs 33px...but now that I think about it maybe it should be 32 

https://github.com/user-attachments/assets/9a53b2f4-36e8-4904-af12-86e6f63aed46